posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -168,7 +168,7 @@ async def minio_client(bucket_name):
 
         yield minio_client
 
-        await delete_all_from_s3(minio_client, bucket_name, key_prefix="/")
+        await delete_all_from_s3(minio_client, bucket_name, key_prefix="")
```

Comment:
the prefix `/` was leaving one file behind that was at the root of the bucket...
so I made it the empty string instead 🤷 

i'm guessing we don't care @Gilbert09 @rossgray

Reply:
if it works 🤷

Reply:
Not sure why that behaviour has changed then but sounds like it should be fine